.github/workflows/release.yml

```diff
@@ -0,0 +1,196 @@
+name: u8timeseries release workflow
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
```

Comment:
should the tests step be also extracted for develop.yml workflow?

Reply:
we have it in both workflows